packages/account-v2/src/containers/PersonalDetailsForm/__tests__/PersonalDetailsForm.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { APIProvider, AuthProvider, useActiveTradingAccount } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import { useCurrentLandingCompany } from '../../../hooks/useCurrentLandingCompany';
+import { PersonalDetailsForm } from '../PersonalDetailsForm';
+
+jest.mock('../PersonalDetails', () => ({
+    ...jest.requireActual('../PersonalDetails'),
+    PersonalDetails: () => <div>Personal Details Container</div>,
+}));
+
+jest.mock('../TaxInformation', () => ({
+    ...jest.requireActual('../TaxInformation'),
+    TaxInformation: () => <div>Tax Information Container</div>,
+}));
+
+jest.mock('../SupportProfessionalClient', () => ({
+    ...jest.requireActual('../SupportProfessionalClient'),
+    SupportProfessionalClient: () => <div>Support Professional Client Container</div>,
+}));
+
+jest.mock('../../../modules/AddressFields', () => ({
+    ...jest.requireActual('../../../modules/AddressFields'),
+    AddressFields: () => <div>Address Fields Container</div>,
```

Comment:
**NIT:** Better to write **Container** tests without mocking the components. This can serve as integration tests. What do u think @amina-deriv @matin-deriv?

Reply:
The component i've mocked have their own unit test case, If we consider integration test, we should write actual integration test as well.